Modal-Dialog/Template.cshtml

```diff
@@ -0,0 +1,226 @@
+  }
+
+  .modalDialog-container {
+    margin: 10px;
+
+  }
+
+  .modalDialog-content {
+    text-align: center;
+    font-size: 16px;
+  }
+
+  .modal-btn-primary {
+    text-align: center;
+    padding: 5px;
+  }
+
+  .primary-style {
+    background-color: #CE3439;
+    border-color: #CE3439;
+    color: white;
+  }
+  .primary-style:hover {
+    background-color: #D65B61 !important; 
+    border-color: #D65B61!important; 
+    color: white;            
+}
+
+.primary-style:active {
+    background-color: #CE3439; 
+    border-color: #CE3439;   
+    color: white;            
+}
+
+.primary-style:visited {
+    background-color: #CE3439; 
+    border-color: #CE3439;   
+    color: white;            
+}
+
+  .modal-btn-secondary {
+    text-align: center;
+    padding: 5px;
+  }
+
+  .secondary-style {
+    background-color: #0069A5;
+    border-color: #0069A5;
+    color: white;
+  }
+  .secondary-style:hover {
+    background-color: #4D91BF!important;
+    border-color: #4D91BF!important;
+    color: white;
+  }
+  .secondary-style:active {
+    background-color: #0069A5;
+    border-color: #0069A5;
+    color: white;
+  }
+  .secondary-style:visited {
+    background-color: #0069A5;
+    border-color: #0069A5;
+    color: white;
+  }
+
+  .emailAccuracy-style{
+    text-align: center;
+    font-size: 16px;
+  }
+
+  .mt-15{
+    margin-top: 15px;
+  }
+
+</style>
+
+@{
+  var moduleID = Dnn.Module.ModuleID;
+  var tabID = Dnn.Module.TabID;
+  var moduleTitle = Model.ModuleTitle;
+  var modalId = Model.ModalId;
+  var modalDialogTitle = Model.ModalDialogTitle;
+  var modalContent = Model.ModalContent;
+  var modalHTMLContent = Model.HTMLContent;
+  var primaryButtonText = Model.PrimaryButtonText;
+  var primaryButtonLink = Model.PrimaryButtonLink;
+  var secondaryButtonText = Model.SecondaryButtonText;
+  var secondaryButtonLink = Model.SecondaryButtonLink;
+  bool IsAuthenticated = User.Identity.IsAuthenticated;
+  var userEmail = User.Identity.IsAuthenticated ? User.Identity.Name : "No user logged in";
+  var cookieSetting = @Model.Settings.UseCookie;
+
+}
+
+<!-- ModalDialog -->
+<div class="modal fade" id="@modalId" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1"
+  aria-labelledby="@modalId Label" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered">
+    <div class="modal-content">
+      <div class="modal-body">
+        <div class="row modal-header-bg-red">
+          <div class="col-xs-12 text-center">
+            <@headerTextType class="modal-body-title text-center modalDialog-title">@modalDialogTitle</@headerTextType>
+          </div>
+        </div>
+        <div class="row col-12 modalDialog-container">
+          <div class="col-12 mt-15">
+            <p class="modalDialog-content">
+              @modalContent
+            </p>
+            @if(IsAuthenticated){
+            <p class="modalDialog-content">
+              <strong>Current email:</strong> @userEmail
```

Comment:
@ArielBlanco1990 Same localization question.

Reply:
@WillStrohl I'll fix it in a moment.